source/known-hook-events.ts

```diff
@@ -1,8 +1,47 @@
-export default [
+import {RequestOptions} from 'https';
+import {IncomingMessage} from 'http';
+
+// TODO: This may be a bit overkill. It was done for different reasons:
```

Comment:
I hope you understand the rationale here, even though it does look quite cumbersome.